src/main/groovy/nebula/plugin/info/reporting/InfoJarManifestPlugin.groovy

```diff
@@ -38,6 +43,39 @@ class InfoJarManifestPlugin implements Plugin<Project>, InfoReporterPlugin {
                     jarTask.manifest.attributes.putAll(attrs)
                 }
             }
+
+            project.afterEvaluate {
+                if(GradleKt.versionGreaterThan(project.gradle, "6.6-rc-1")) {
+                    configureMetaInfNormalization(project)
+                }
+            }
+
         }
     }
+
+    @CompileDynamic
+    private void configureMetaInfNormalization(Project project) {
+        InfoBrokerPluginExtension extension = project.extensions.getByName('infoBroker') as InfoBrokerPluginExtension
+        project.normalization.runtimeClasspath.metaInf(new Action<MetaInfNormalization>() {
+            @Override
+            void execute(MetaInfNormalization metaInfNormalization) {
+                if(extension.ignoreManifestForNormalization.isPresent() && extension.ignoreManifestForNormalization.get()) {
+                    metaInfNormalization.ignoreManifest()
+                }
+                if(extension.ignoreNormalizationCompletely.isPresent() && extension.ignoreNormalizationCompletely.get()) {
+                    metaInfNormalization.ignoreCompletely()
+                }
+                if(extension.ignoredPropertiesForNormalization.isPresent()) {
+                    extension.ignoredPropertiesForNormalization.get().each { manifestProperty ->
```

Comment:
We know what the default changing properties are here - think we should configure those explicitly rather than exposing general purpose normalization extensions.